### DISCUSSION TITLE
Licence usage in dev

### DISCUSSION THREAD

**Discussion by jjakucionis** [sachinchoolur/lightGallery#1391] [Q&A]
I have bought a lightgallery personal licence and I have questions about using it on a development website first.
For example, now I am developing my website with URL 'websitename.development.com', later when I push it to production, URL will be 'websitename.com'.
Will there be any problems with using the same licence key? Dev website later will be deleted

**Comment by sachinchoolur** [comment]
Hey @jjakucionis,

No problem at all. You can use the same license key for both production and development versions
